src/MOI/MOI_wrapper.jl

```diff
@@ -2799,27 +2783,16 @@ function _ensure_conflict_computed(model::Optimizer)
     end
 end
 
-"""
-    ConflictStatus()
+MOI.supports(::Optimizer, ::MOI.ConflictStatus) = true
 
-Return an `MOI.TerminationStatusCode` indicating the status of the last
-computed conflict. If a minimal conflict is found, it will return
-`MOI.OPTIMAL`. If the problem is feasible, it will return `MOI.INFEASIBLE`. If
-`compute_conflict` has not been called yet, it will return
-`MOI.OPTIMIZE_NOT_CALLED`.
-"""
-struct ConflictStatus <: MOI.AbstractModelAttribute  end
-
-MOI.is_set_by_optimize(::ConflictStatus) = true
-
-function MOI.get(model::Optimizer, ::ConflictStatus)
+function MOI.get(model::Optimizer, ::MOI.ConflictStatus)
     if model.conflict === nothing
-        return MOI.OPTIMIZE_NOT_CALLED
+        return MOI.COMPUTE_CONFLICT_NOT_CALLED
     elseif model.conflict.stat == CPX_STAT_CONFLICT_MINIMAL
-        return MOI.OPTIMAL
+        return MOI.CONFLICT_FOUND
     elseif model.conflict.stat == CPX_STAT_CONFLICT_FEASIBLE
-        return MOI.INFEASIBLE
-    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_CONTRADICTION
+        return MOI.NO_CONFLICT_EXISTS
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_CONTRADICTION # TODO: these codes do not exist in MOI.
         return MOI.OTHER_LIMIT
```

Comment:
The return type of this function isn't type-stable. Maybe we need a `MOI.COMPUTE_CONFLICT_FAILED` status. I don't think we need to break out all the possible limits.

Reply:
I added a few status codes in https://github.com/JuliaOpt/MathOptInterface.jl/pull/1091, without delving into the details.